### PR TITLE
fix: check bot account by bot_id when message is recieved

### PR DESCRIPTION
**Pull request by rrrrs09**
# Description

Please include a summary of the change.

# Checklist

- [x] Linters and tests have passed
- [ ] I've written a changelog for PR change
- [ ] I'll make a release when PR is approved

### CODE REVIEW

File: tests/test_end_to_end.py [hunk 1]
```diff
@@ -59,6 +60,8 @@ async def command_handler(request: Request) -> JSONResponse:
             build_bot_disabled_response(error_label),
             status_code=HTTPStatus.SERVICE_UNAVAILABLE,
         )
+    except UnknownBotAccountError:
+        pass
```
Comment (Nimond) [inline tests/test_end_to_end.py:64]: Should return 503

File: botx/bot/bot.py [hunk 2]
```diff
@@ -145,11 +145,17 @@ def async_execute_raw_bot_command(self, raw_bot_command: Dict[str, Any]) -> None
...
 
     def async_execute_bot_command(self, bot_command: BotCommand) -> None:
+        self.ensure_bot_id_exists(bot_command.bot_id)
+
         task = asyncio.create_task(
             self._handler_collector.handle_bot_command(bot_command, self),
         )
         self._tasks.add(task)
 
+    def ensure_bot_id_exists(self, bot_id: UUID) -> None:
```
Comment (Nimond) [inline botx/bot/bot.py:155]: Are you sure this method must be public?

File: tests/test_end_to_end.py (hunk 1 above)
Comment (Nimond) [reply tests/test_end_to_end.py:64]: Also write end_to_end test for it

File: botx/bot/bot.py (hunk 2 above)
Comment (lxmnk) [reply botx/bot/bot.py:155]: This method should be public, but in `BotAccountsStorage`, not in `Bot`.

File: tests/test_end_to_end.py (hunk 1 above)
Comment (rrrrs09) [reply tests/test_end_to_end.py]: fixed

File: botx/bot/bot.py (hunk 2 above)
Comment (rrrrs09) [reply botx/bot/bot.py]: done

**Review by lxmnk — APPROVED** [review]

File: tests/test_end_to_end.py [hunk 3]
```diff
@@ -167,6 +178,55 @@ def test__web_app__bot_command(
     assert response.status_code == HTTPStatus.ACCEPTED
 
 
+def test__web_app__unknown_bot(
```
Comment (Nimond) [inline tests/test_end_to_end.py:181]: `test__web_app__unknown_bot` -> `test__web_app__unknown_bot_response`

**Review by Nimond — APPROVED** [review]

File: tests/test_end_to_end.py (hunk 3 above)
Comment (rrrrs09) [reply tests/test_end_to_end.py]: done

**Pull request merged**